invenio_rdm_records/services/config.py

```diff
@@ -114,16 +114,16 @@ class RDMUserRecordsServiceConfig(RDMRecordServiceConfig):
             # 'languages': {
             #     'terms': {'field': 'metadata.languages.id'},
             # },
-            'status': {
-                'terms': {'field': 'status'},
+            'published': {
```

Comment:
Question: We could leave it as ``status`` so that the query is ``?status=published`` and ``?status=new``

Reply:
We cannot do that because the actual filter would be `status=true/false` as the `is_published` is a boolean.... The published / new labels will be applied from the UI serializer...